docs/connection.rst

```diff
@@ -57,9 +57,9 @@ use the stub implementation of CentralEGA and the following queues, per
 +=================+=====================================+
 | files           | Trigger for file ingestion          |
 +-----------------+-------------------------------------+
-| archived        | The file is in the vault            |
+| archived        | The file is in the archive            |
```

Comment:
this is a malformed table, see error from Sphinx:
```
/home/stenegru/csc/dev/EGA-LocalEGA/docs/connection.rst:55: WARNING: Malformed table.

+-----------------+-------------------------------------+
| Name            | Purpose                             |
+=================+=====================================+
| files           | Trigger for file ingestion          |
+-----------------+-------------------------------------+
| archived        | The file is in the archive            |
+-----------------+-------------------------------------+
| qc              | The file is "verified" in the archive |
|                 | and Quality Controllers can execute |
+-----------------+-------------------------------------+
```

to replicate this (must have `sphinx` and `sphinx_rtd_theme` installed):
```
cd docs
make html
```

should look like:
```
+-----------------+---------------------------------------+
| Name            | Purpose                               |
+=================+=======================================+
| files           | Trigger for file ingestion            |
+-----------------+---------------------------------------+
| archived        | The file is in the archive            |
+-----------------+---------------------------------------+
| qc              | The file is "verified" in the archive |
|                 | and Quality Controllers can execute   |
+-----------------+---------------------------------------+
```